informal_update/models.py

```diff
@@ -0,0 +1,219 @@
+    )
+    district = models.ForeignKey(
+        District, verbose_name=_('district'), on_delete=models.CASCADE,
+        related_name='informal_district'
+    )
+    client_id = models.CharField(max_length=50, null=True, blank=True)
+
+    class Meta:
+        unique_together = ('informal_update', 'country')
+        verbose_name = _('informal country district')
+        verbose_name_plural = _('informal countries districts')
+
+    def __str__(self):
+        return f'{self.country} - {self.district}'
+
+
+class InformalAction(models.Model):
+    """ Action taken for Informal Update """
+
+    name = models.CharField(verbose_name=_('name'), max_length=400)
+    organizations = ArrayField(
+        models.CharField(choices=ActionOrg.CHOICES, max_length=4),
+        verbose_name=_('organizations'), default=list, blank=True
+    )
+    informal_update_types = ArrayField(
+        models.CharField(choices=ActionType.CHOICES, max_length=16),
+        verbose_name=_('informal update types'), default=list,
+    )
+    category = models.CharField(
+        max_length=255, verbose_name=_('category'), choices=ActionCategory.CHOICES, default=ActionCategory.GENERAL
+    )
+    is_disabled = models.BooleanField(verbose_name=_('is disabled?'), default=False, help_text=_('Disable in form'))
+    tooltip_text = models.TextField(verbose_name=_('tooltip text'), null=True, blank='true')
+    client_id = models.CharField(max_length=50, null=True, blank=True)
+
+    class Meta:
+        verbose_name = _('informal action')
+        verbose_name_plural = _('informal actions')
+
+    def __str__(self):
+        return self.name
+
+
+class InformalActionsTaken(models.Model):
+    """ All the actions taken by an organization in Informal Update """
+
+    organization = models.CharField(
+        choices=ActionOrg.CHOICES,
+        verbose_name=_('organization'), max_length=16,
+    )
+    actions = models.ManyToManyField(InformalAction, verbose_name=_('actions'), blank=True)
+    summary = models.TextField(verbose_name=_('summary'), null=True, blank=True)
+    informal_update = models.ForeignKey(
+        InformalUpdate, verbose_name=_('informal update'), related_name='actions_taken_informal', on_delete=models.CASCADE
+    )
+    client_id = models.CharField(max_length=50, null=True, blank=True)
+
+    class Meta:
+        verbose_name = _('actions taken informal')
+        verbose_name_plural = _('all actions taken informal')
+
+    def __str__(self):
+        return f'{self.organization} - {self.actions}'
+
+
+class InformalEmailSubscriptions(models.Model):
+    share_with = models.CharField(
+        max_length=50, choices=InformalUpdate.InformalShareWith.choices,
+        default=InformalUpdate.InformalShareWith.IFRC_SECRETARIAT,
+        verbose_name=_('share with')
+    )
+    group = models.ForeignKey(Group, on_delete=models.CASCADE, null=True, blank=True, related_name='informal_email_subscription')
+
+    def __str__(self):
+        return self.share_with
+
```

Comment:
Let's add a validation to avoid selecting RCRC_NETWORK_AND_DONOR here.
```python3
from django.core.exceptions import ValidationError

....

    def clean(self):
        if self.share_with == InformalUpdate.InformalShareWith.RCRC_NETWORK_AND_DONOR:
            raise ValidationError(ugettext('RCRC Network and Donor is not allowed. Use Donor table for donors'))
        return super().clean()
```

Reply:
According to this https://github.com/IFRCGo/go-api/pull/1261#discussion_r808832435

It is not required.